source/content/guides/decoupled-sites/site.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Pantheon Decoupled
+subtitle: 
+description: 
+categories: [platform]
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan]
+reviewed: "2022-07-31"
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled-sites/site
+anchorid: site
+---
+
+# Before You Begin
+
+You should have access to the Early Access Customer checklist provided during onboarding. To begin using Pantheon Decoupled, ensure you meet the following prerequisites:
+
+* Your repository is in GitHub.
+
+* You have a domain to use for the decoupled site.
+
+* You will use Next.js or Gatsby as the Frontend framework.
+
+* (Optional) You are using Drupal 9 or WordPress for the CMS.
+
+* (Optional) If using a preconfigured starter kit template, you will use one of the following combinations:
+
+    * Drupal 9 and Next.js
+
+    * WordPress and Gatsby
+
+    Note: Other configurations are not currently supported. 
+
+
+
+## Site Setup
+
+Pantheon Decoupled is available through the New Dashboard. To access it log in to your Pantheon Dashboard and Navigate to the Sites page. 
+
+. Select **Sites** in the the New Dashboard. 
+
+. Click the **Decoupled** tab to view the Early Access offering. 
+
+. Click **+Create New Site** in the New Dashboard  under the **Decoupled** tab to begin your decoupled journey. 
+
+
+You can create an application from scratch using **Importing a repository** or you can use our starter kit template for:
+
+* Server side render applications: Next.js and Drupal
+
+* Static site generated sigte: Gatsby and WordPress
+
+Under the Decoupled Site section on the Site Creation page, click the option that best suits your needs. Ensure you are using compatible versions for the associated tooling before you proceed (e.g. Drupal 9, Next.js v12, etc. ) Refer to the section on compatibility for more information.
+
+# Connect your Account
+
+. To start you will need to specify the GitHub account associated with your decoupled site. After you select the decoupled site option that best suits your needs, you will be prompted to connect your Git provider. 
+
+. Select the GitHub option and click Connect. A window for GitHub.com is displayed, and you are prompted “Where do you want to install Pantheon”. 
```

Comment:
Add a warning here that if they're trying to install on a GitHub organization, they need to be an admin of that org on GitHub or they will get an error, and that on prem GitHub enterprise orgs are not supported (need to confirm this).